charts/ghga-common/templates/_serviceaccount.tpl

```diff
@@ -0,0 +1,11 @@
+{{- define "ghga-common.serviceaccount" -}}
+{{- if .Values.serviceaccount.create }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
+  labels:
```

Comment:
Maybe add here also the standard labels from bitnami library chart:
`{{- include "common.labels.standard" . | nindent 4 }}`